doc/getting_started/installation_linux.rst

```diff
@@ -99,6 +99,12 @@ On Arch:
    sudo pacman -S git cmake ninja gperf ccache dfu-util dtc wget \
        python-pip python-setuptools python-wheel xz file make
 
+On Arch, you also need the following packages to flash a board from the AUR. The install differs, depending on the tool you use to access the AUR.
+
+.. code-block:: console
+   python-pykwalify
+   pyocd
```

Comment:
These are both installed by the pip install ... requirements.txt file, no?

Reply:
They should be. But at min 2 Arch Laptops, it did not happen.

Reply:
these two python packages are indeed in the requirements.txt file so don't need to be installed separately.

Reply:
> They should be. But at min 2 Arch Laptops, it did not happen.

Do we have some way of reproducing this?  It seems odd that the pip command wouldn't install the python packages.  @Allegra42 should we open an issue for this?  I'm kinda with @dbkinder that updating the docs is probably not the right approach.